app/assets/javascripts/components/static/about_page.es6.jsx

```diff
@@ -2,69 +2,85 @@ class AboutPage extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      people: [
-        'daniel', 'richard', 'rocky',
-        'rachel', 'sahil', 'lancy'
-      ],
-    }
-  }
+    this.individuals = [
```

Comment:
we can move this into a new file under constants. maybe call the file individual_constants.es6.jsx. Check out the constants files for an example (it creates a global variable thats accessible in any file!)